templates/execution/token.html

```diff
@@ -245,14 +245,14 @@
       <h1 class="h4 mb-1 mb-md-0 header-token text-truncate">
         <div class="my-md-3 font-weight-bold">
           <span class="d-flex">
-            <span class="mr-1 d-flex align-items-center">
+            <span class="mr-1 d-flex align-items-center" title="{{ .Data.Metadata.Symbol | formatTokenSymbolTitle }}">
```

Comment:
There is no need to use the pipe symbol '|' here. Instead, writing
`{{ formatTokenSymbolTitle .Data.Metadata.Symbol }}`
would achieve the same but be more readable imho.

This applies to more of your changes.

Reply:
I slightly prefer the pipeline because if I see the '|' character I immediately know that a function is getting called.
It's already used like this with only one parameter and one function in some files but if you want it changed we can talk about it.

Reply:
We now also want the replacement text to be [hidden-symbol].
For this review please also keep in mind that I show the tooltip also on mouseover of the logo (if it has one).
I think this makes sense as both together form a href, see e.g. here:
https://beaconcha.in/address/0xa4580ea641b98a25f3549e2712415028f8178824#erc20Txns